runtime/gc_base/GCExtensions.hpp

```diff
@@ -290,6 +298,20 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	MMINLINE MM_ContinuationObjectList* getContinuationObjectLists() { return continuationObjectLists; }
 	MMINLINE void setContinuationObjectLists(MM_ContinuationObjectList* newContinuationObjectLists) { continuationObjectLists = newContinuationObjectLists; }
 
+	static void releaseNativesForContinuationObject(MM_EnvironmentBase* env, j9object_t objectPtr)
+	{
+#if JAVA_SPEC_VERSION >= 19
+		J9VMThread *vmThread = (J9VMThread *)env->getLanguageVMThread();
+
+		if (verify_continuation_list == continuationListOption) {
+		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
+		Assert_MM_true(NULL == continuation);
```

Comment:
indentation

Reply:
I think we should not assertions to this `.hpp` file. We either move helper to `.cpp` file or create special slow path helper for assertion only in `.cpp`

Reply:
or remove assertion obviously

Reply:
I have no problem keeping it here, but don't know if compiler will have

Reply:
you have particular concern since this is one of top level *hpp files?

Reply:
yes, we had this problem in the past. Any place it is included it should have proper assertions setup. Currently we have no assertions in this file.

Reply:
could you be more specific - we have asserts in some other *hpp files

Reply:
Adding assertions to `.hpp` file without proper configuring them in this `.hpp` file creates hidden assumption (limitation) for environment this method is going to be called - the assertions should be configured there. An attempt to include `.h` required for assertions in this file would cause multiple compilation errors across code